generators/chipyard/src/main/resources/csrc/cospike.cc

```diff
@@ -87,7 +94,7 @@ extern "C" void cospike_cosim(long long int cycle,
                     nullptr,
                     info->isa.c_str(),
                     "MSU",
-                    "vlen:128,elen:64",
+                    "vlen:512,elen:64",
```

Comment:
Remove this for now, we need a API to pass from thhe chisel the vlen (you can implement this in this PR if you want)